pages/login.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { ChangeEvent, useState } from 'react';
+import PageLayout from "../components/layout/PageLayout";
+import styles from "../styles/pages/register.module.scss";
+import TextField from '@mui/material/TextField';
+import Select, { SelectChangeEvent } from '@mui/material/Select';
+import { BiErrorCircle } from "react-icons/bi"
+import ErrorTextBox from '../components/register/ErrorTextBox';
+import { IconButton, InputAdornment } from '@mui/material';
+import { AiFillEye, AiFillEyeInvisible } from "react-icons/ai"
+
+function Login() {
+
+    const [formValues, setFormValues] = useState({
+        username: '',
+        password: '',
+    })
+
+    const [error, setError] = useState(false);
+    const [showPassword, setShowPassword] = useState(false);
+
+    const handleChange = (event: ChangeEvent<HTMLInputElement>
+        | SelectChangeEvent
+        | React.ChangeEvent<HTMLTextAreaElement
+            | HTMLInputElement>) => {
+        event.preventDefault();
+        const name = event.target.name;
+        const value = event.target.value;
+        setFormValues({
+            ...formValues,
+            [name]: value
+        })
+    }
+
+    const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+        event.preventDefault();
+
+        // check if login credentials are valid
+        if (true) {
+            setError(true);
+
+            setTimeout(() => {
+                setError(false);
+            }, 3000);
```

Comment:
@Ishwarendra has made a separate dialog flow to show errors, integrate that here.
You can refer #4